src/main/target/AOCODARCF4V3/target.h

```diff
@@ -19,8 +19,13 @@
 
 #define USE_TARGET_CONFIG
 
-#define TARGET_BOARD_IDENTIFIER         "AOF4V3"
-#define USBD_PRODUCT_STRING             "AocodaRCF4V3"
+#ifdef AOCODARCF4V3
+#define TARGET_BOARD_IDENTIFIER "AOF4V3F"
```

Comment:
Just AOF4V3 for flash version, AOF4V3SD for SDCard version.

Reply:
Change AOF4V3F to AOF4V3